contracts/src/v0.8/dev/ArbitrumValidator.sol

```diff
@@ -247,7 +247,7 @@ contract ArbitrumValidator is TypeAndVersionInterface, AggregatorValidatorInterf
       s_gasConfig.refundableAddress, // callValueRefundAddress
       L2_GAS_LIMIT,
       s_gasConfig.maxGasPrice,
-      currentAnswer == 0 ? CALL_RAISE_FLAG : CALL_LOWER_FLAG
+      currentAnswer == 0 ? CALL_LOWER_FLAG : CALL_RAISE_FLAG
```

Comment:
I might be missing some context here, but why is this logic reversed?

Reply:
https://github.com/smartcontractkit/chainlink/pull/4641#discussion_r667793907

Reply:
So initially we implemented:

- L1 “seq_online” data point (feed)
- L2 “seq_offline” data point (flag)

Then we changed it to:

- L1 “seq_offline” data point (feed)
- L2 “seq_offline” data point (flag)

It made more sense to me that both L1 & L2 data point represent the same thing, not opposites.

The options are:

- L1 “seq_online” -> L2 “seq_online”
- L1 “seq_online” -> L2 “seq_offline”
- L1 “seq_offline” -> L2 “seq_offline”
- L1 “seq_offline” -> L2 “seq_online” (this one makes the least sense IMHO)

@yosriady what makes most sense to you?